packages/gatsby/src/query/queue.ts

```diff
@@ -1,46 +1,61 @@
-const Queue = require(`better-queue`)
-const { store } = require(`../redux`)
-const FastMemoryStore = require(`../query/better-queue-custom-store`)
-const { queryRunner } = require(`../query/query-runner`)
-const { websocketManager } = require(`../utils/websocket-manager`)
-const { GraphQLRunner } = require(`./graphql-runner`)
-
-const createBaseOptions = () => {
+import Queue from "better-queue"
+import { store } from "../redux"
+import { MemoryStoreWithPriorityBuckets } from "../query/better-queue-custom-store"
+import { queryRunner } from "../query/query-runner"
+import { websocketManager } from "../utils/websocket-manager"
+import { GraphQLRunner } from "./graphql-runner"
+import BetterQueue from "better-queue"
+import { IExecutionResult } from "./types"
+import { ProgressActivityTracker } from "../.."
+
+export type Task = any
+type TaskResult = any
```

Comment:
I couldn't locate a better type, but at least we can just change these lines if one exists.